### ISSUE TITLE
Issues with React 18.2.0

### ISSUE THREAD

**Issue by byrnesr1975** [steima/react-qr-barcode-scanner#2]
Hi,

This component looks like it will really help me out, but I've just tried to install and got these errors, are you able to assist?

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: ReactApp@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from @steima/react-qr-barcode-scanner@1.1.2
npm ERR! node_modules/@steima/react-qr-barcode-scanner
npm ERR!   @steima/react-qr-barcode-scanner@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\simon_000\AppData\Local\npm-cache\eresolve-report.txt for a full report.

**Comment by steima** [comment]
Is this still relevant?

**Comment by byrnesr1975** [comment]
Yes, this is still relevant for me, it would really help if I could get this working.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by steima** [comment]
OK, perhaps I can test with React18 on the weekend.

**Comment by byrnesr1975** [comment]
Thank you so much, any assistance you could provide would be very much appreciated.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by CarvalhoFilipe** [comment]
I also need it, if possible see for version 18.0, thank you very much!!

**Comment by argiweb** [comment]
how many weekend you need?

**Comment by akashurrehman** [comment]
> how many weekends you need?

I can resolve this issue in 2-3 hours. Let me know if still you are facing this issue
